File: recursion/python/codeup_1920.py
result = ''
def decimal_to_binary(n):
    global result # 뭔가 이상한데....
    if n == 0 or n == 1:
        result = str(n) + result
        return result
    else:
        result = str(n%2) + result
        return decimal_to_binary(n//2)

print(decimal_to_binary(10))

File: recursion/java/Codeup1920.java
import java.util.Scanner;

public class Codeup1920 {

    static String result = "";

    public static String decimalToBinary(int n) {
        if (n == 0 || n == 1) {
            result = Integer.toString(n) + result;
            return result;
        } else {
            result = Integer.toString(n%2) + result;
            return decimalToBinary(n/2);
        }
    }

    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int i = s.nextInt();
        System.out.println(decimalToBinary(i));
    }
}
